src/main/kotlin/nl/avisi/structurizr/site/generatr/site/views/CDN.kt

```diff
@@ -1,63 +1,65 @@
 
-        fun bulmaCss() = dependencies.single { it.name == "bulma" }.let {
-            "${it.baseUrl()}/css/bulma.min.css"
-        }
+    fun bulmaCss() = dependencies.single { it.name == "bulma" }.let {
+        "${it.baseUrl()}/css/bulma.min.css"
+    }
 
-        fun katexJs() = dependencies.single { it.name == "katex" }.let {
-            "${it.baseUrl()}/dist/katex.min.js"
-        }
+    fun katexJs() = dependencies.single { it.name == "katex" }.let {
+        "${it.baseUrl()}/dist/katex.min.js"
+    }
 
-        fun katexCss() = dependencies.single { it.name == "katex" }.let {
-            "${it.baseUrl()}/dist/katex.min.css"
-        }
+    fun katexCss() = dependencies.single { it.name == "katex" }.let {
+        "${it.baseUrl()}/dist/katex.min.css"
+    }
 
-        fun lunrJs() = dependencies.single { it.name == "lunr" }.let {
-            "${it.baseUrl()}/lunr.min.js"
-        }
+    fun lunrJs() = dependencies.single { it.name == "lunr" }.let {
+        "${it.baseUrl()}/lunr.min.js"
+    }
 
-        fun lunrLanguagesStemmerJs() = dependencies.single { it.name == "lunr-languages" }.let {
-            "${it.baseUrl()}/min/lunr.stemmer.support.min.js"
-        }
+    fun lunrLanguagesStemmerJs() = dependencies.single { it.name == "lunr-languages" }.let {
+        "${it.baseUrl()}/min/lunr.stemmer.support.min.js"
+    }
 
-        fun lunrLanguagesJs(language: String) = dependencies.single { it.name == "lunr-languages" }.let {
-            "${it.baseUrl()}/min/lunr.$language.min.js"
-        }
+    fun lunrLanguagesJs(language: String) = dependencies.single { it.name == "lunr-languages" }.let {
+        "${it.baseUrl()}/min/lunr.$language.min.js"
+    }
 
-        fun mermaidJs() = dependencies.single { it.name == "mermaid" }.let {
-            "${it.baseUrl()}/dist/mermaid.esm.min.mjs"
-        }
+    fun mermaidJs() = dependencies.single { it.name == "mermaid" }.let {
+        "${it.baseUrl()}/dist/mermaid.esm.min.mjs"
+    }
 
-        fun svgpanzoomJs() = dependencies.single { it.name == "svg-pan-zoom" }.let {
-            "${it.baseUrl()}/dist/svg-pan-zoom.min.js"
-        }
+    fun svgpanzoomJs() = dependencies.single { it.name == "svg-pan-zoom" }.let {
+        "${it.baseUrl()}/dist/svg-pan-zoom.min.js"
+    }
 
-        fun webfontloaderJs() = dependencies.single { it.name == "webfontloader" }.let {
-            "${it.baseUrl()}/webfontloader.js"
-        }
+    fun webfontloaderJs() = dependencies.single { it.name == "webfontloader" }.let {
+        "${it.baseUrl()}/webfontloader.js"
     }
 
     @Serializable
-    data class Dependency(val name: String, val version: String) {
-        fun baseUrl() = "$CDN_BASE$name@$version"
+    data class Dependency(val cdnBaseURL: String, val name: String, val version: String) {
+        fun baseUrl() = when (cdnBaseURL.last().toString()) {
+                        "/" -> "$cdnBaseURL$name@$version"
```

Comment:
Is is saner to strip ending slashes directly after reading the value from comfiguration, so something like:

```kotlin
private val cdnBaseURL = workspace.views.configuration.properties
    .getOrDefault("generatr.site.cdn", "https://cdn.jsdelivr.net/npm")
    .trimEnd('/')
```